gsy_framework/sim_results/cumulative_grid_trades.py

```diff
@@ -192,11 +192,11 @@ def _update_area_children_in_accumulated_trades_dict(self, accumulated_trades, a
         Returns: None
         """
         current_child_uuids = [
-            c["uuid"] for c in accumulated_trades[area["uuid"]]["children"]
+            c["uuid"] for c in accumulated_trades[area["uuid"]].get("children", [])
         ]
         for child in area.get("children", []):
             if child["uuid"] not in current_child_uuids:
-                accumulated_trades[area["uuid"]]["children"].append(
+                accumulated_trades[area["uuid"]].setdefault("children", []).append(
```

Comment:
Sorry, but I also need some info here. Why is this needed? For the case when `children` is not part of the `accumulated_trades[area["uuid"]]` ?

Reply:
We had a `KeyError: 'children'` for `accumulated_trades[area["uuid"]]["children"]`.  Which means that the `accumulated_trades` has an area which is missing `children` key. So the append method will raise an error in such case. That's why I create the `children` list and append to it, and if it's already created then it will use existed one.

Reply:
Here, https://github.com/gridsingularity/gsy-framework/pull/343#discussion_r861098534 also applies.